src/EventManagement.Web/Pages/Admin/Events/Edit.cshtml.cs

```diff
@@ -46,16 +47,7 @@ public EditModel(ApplicationDbContext context)
                 return Page();
             }
 
-            _context.Attach(EventInfo).State = EntityState.Modified;
```

Comment:
@losviko any specific reason you chose to use this over `_context.EventInfos.Update(EventInfo)`?

Reply:
It seems to generate smoother sql statements with only the changed fields getting updated, but I have no feelings here :)